src/event.rs

```diff
@@ -643,6 +757,31 @@ pub enum KeyCode {
     Null,
     /// Escape key.
     Esc,
+    /// Caps Lock key.
+    ///
+    /// **Note:** this and all following keys can only be read if
```

Comment:
I prefer a redundancy here. If you will please copy and paste the same text for each key to which this comment is applicable to. It's just to be clear towards the user..

Reply:
If we update the documentation we'll have to do find replace, but at least we are clear.

Reply:
Would be awesome to see if windows supports those key combinations

Reply:
<html>
<body>
<!--StartFragment-->

VK_SNAPSHOT | 0x2C | PRINT SCREEN key
-- | -- | --


<!--EndFragment-->
</body>
</html>
<html>
<body>
<!--StartFragment-->

VK_CAPITAL | 0x14 | CAPS LOCK key
-- | -- | --


<!--EndFragment-->
</body>
</html>

<html>
<body>
<!--StartFragment-->

VK_NUMLOCK | 0x90 | NUM LOCK key
-- | -- | --
VK_SCROLL | 0x91 | SCROLL LOCK key

<!--EndFragment-->
</body>
</html>

https://docs.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes

Actually pretty easy. I might implement that after this pr is merged.